elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -0,0 +1,426 @@
+import prairielearn as pl
+import lxml.html
+from html import escape
+import numpy as np
+import math
+import chevron
+import random
+
+
+def prepare(element_html, element_index, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    required_attribs = ['answers-name']
+    optional_attribs = ['weight', 'label', 'comparison', 'rtol', 'atol', 'digits', 'format-type', 'allow-partial-credit', 'allow-feedback']
+    pl.check_attribs(element, required_attribs, optional_attribs)
+
+
+def render(element_html, element_index, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    # get the name of the element, in this case, the name of the array
+    name = pl.get_string_attrib(element, 'answers-name')
+    label = pl.get_string_attrib(element, 'label', None)
+    format_type = pl.get_string_attrib(element, 'format-type', 'latex')
```

Comment:
If it is an option to display both the correct answer and the submitted answer in python format, it might be best to display these answers in both python **and** matlab format, as in `pl-matrix-output`, where the student can switch back and forth between these two.

Would it be best to avoid the entire issue and display each answer in a table? (With the same code that is used to create the array of inputs in the question panel.)